packages/iov-keycontrol/src/wallets/slip10wallet.ts

```diff
@@ -307,7 +307,10 @@ export class Slip10Wallet implements Wallet {
         //   const r = sig.r();
         //   const s = sig.s();
         //   const recoveryParam = sig.recovery;
-        signature = (await Secp256k1.createSignature(message, privkey)).toFixedLength();
+        signature =
+          prehashType === PrehashType.Keccak256
+            ? (await Secp256k1.createSignature(message, privkey)).toFixedLength()
+            : (await Secp256k1.createSignature(message, privkey)).toDer();
```

Comment:
Please revert that change. Everyone gets fix length encoding. There is no good reason to default to DER. When you need DER, you can convert fixed length -> DER as described in the comment.